scripts/qcow.sh

```diff
@@ -66,6 +66,7 @@ install_packages() (
     chroot_exec apt purge -y ubuntu-advantage-tools ubuntu-cloud-minimal ubuntu-drivers-common ubuntu-release-upgrader-core unattended-upgrades xz-utils
 
     chroot_exec apt autoremove -y
+    chroot_exec apt upgrade -y
```

Comment:
Maybe we can move this a bit earlier, I suspect there may be some leftover packages after the upgrade.